### PR TITLE
Added graphql query linting via existing eslint script.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
Installed the Apollo graphql plugin for eslint, and told eslint to use it (and how to find the schema).

Also told eslint to check .tsx files, but not .ts files because we need to upgrade before it can do that (see typescript-eslint/typescript-eslint#2446).

I just enabled the basic query validation rule, but there are [several other options available](https://github.com/apollographql/eslint-plugin-graphql#selecting-validation-rules) (along with some configuration tweaks that could be applied to this one).

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
Pushed back to draft because I realized that to make this actually awesome, I should make the `eslint` job run on graphql schema changes as well, which I will do... tomorrow.

**Comment by benwarfield-usds** [comment]
Demonstration of this failing checks on a back-end branch: https://github.com/CDCgov/prime-data-input-client/actions/runs/415916596.

The checks that are failing here are also failing on master, which is... not great.

File: frontend/package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "build": "react-scripts build",
     "postbuild": "echo 'pushstate: enabled' > build/Staticfile",
     "lint": "npm-run-all -p lint:eslint lint:prettier",
-    "lint:eslint": "eslint --quiet --ext .js --ext .jsx ./src",
+    "lint:eslint": "eslint --quiet --ext .js --ext .jsx --ext .tsx ./src",
```
Comment (timothybest-usds) [inline frontend/package.json:53]: we should add `--ext .ts` to this as well

**Comment by dmethvin-gov** [comment]
This should give us feedback in VSCode too which would be stellar

**Review by timothybest-usds — APPROVED** [review]

File: frontend/package.json (hunk 1 above)
Comment (dmethvin-gov) [reply frontend/package.json:53]: Should we do the TS upgrade in this PR?

Comment (dmethvin-gov) [reply frontend/package.json:53]: I'd suggest no, I had a few speed bumps doing this in a previous project so I'd just merge this as-is and we can make a ticket to do the upgrade and add `--ext .ts`

Comment (benwarfield-usds) [reply frontend/package.json:53]: I have a separate branch already pending with this change, but it didn't seem like a good idea to make this depend on that. :-)

**Comment by benwarfield-usds** [comment]
I can confirm that it does give feedback in VSC! I was unreasonably excited by this.

**Pull request merged**